include/common/firmware/can.h

```diff
@@ -7,6 +7,12 @@
 extern "C" {
 #endif  // __cplusplus
 
+
+/**
+ * The global can 1 handle.
+ */
+extern FDCAN_HandleTypeDef fdcan1;
```

Comment:
this doesn't feel like a great idea. i would hope we can really strictly limit where we use globals like this and i think adding some friction to using it by requiring it to be declared extern everywhere it's used will help.